src/Query/Select.php

```diff
@@ -46,14 +46,24 @@ protected static function validate(Order ...$orders)
     }
 
     /**
-     * @return null|Where
+     * @return ConditionGroup[]
      */
     public function where()
     {
         return $this->where;
     }
 
     /**
+     * An alias of where().
+     *
+     * @return ConditionGroup[]
+     */
+    public function conditionGroups()
```

Comment:
Why aliasing?

Reply:
We can't easily assume that `where()` returns an array. Longer but clearer alias may be useful for newcomer.

Reply:
Let's say a newcomer looks at a list of completions. These duplicate entries look like there are certain differences between the methods: the same return-type but different names. I am opposed to this unnecessary alias.

Reply:
Okay